packages/theme/css/global.css

```diff
@@ -17,15 +17,15 @@ body {
   font-family: "uitk-sans";
 }
 
-.uitk-theme {
+.uitk-provider {
```

Comment:
Thoughts on moving this to ToolkitProvider.css and unifying it with how other components are treated.